tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -367,5 +367,70 @@ public void BuildsAppAfterInstallingAllRequiredPlatforms()
                 },
                 result.GetDebugInfo());
         }
+
+        [Fact]
+        public void BuildsApplication_ByDynamicallyInstallingSDKs_IntoCustomDynamicInstallationDir()
+        {
```

Comment:
can we add one test for python and php as well?

Reply:
Sure, will do